specification/cognitiveservices/data-plane/AutoSuggest/stable/v1.0/examples/SuccessfulQueryRequest.json

```diff
@@ -15,61 +15,61 @@
         },
         "suggestionGroups": [
           {
-            "_type": "Suggestions\/SuggestionGroup",
+            "_type": "Suggestions/SuggestionGroup",
             "name": "Web",
             "searchSuggestions": [
               {
                 "_type": "SearchAction",
-                "url": "https:\/\/www.bing.com\/search?q=amazon&FORM=USBAPI",
+                "url": "https://www.bing.com/search?q=amazon&FORM=USBAPI",
                 "displayText": "amazon",
                 "query": "amazon",
                 "searchKind": "WebSearch"
               },
               {
                 "_type": "SearchAction",
-                "url": "https:\/\/www.bing.com\/search?q=alaska+airlines&FORM=USBAPI",
+                "url": "https://www.bing.com/search?q=alaska+airlines&FORM=USBAPI",
                 "displayText": "alaska airlines",
                 "query": "alaska airlines",
                 "searchKind": "WebSearch"
               },
               {
                 "_type": "SearchAction",
-                "url": "https:\/\/www.bing.com\/search?q=aol&FORM=USBAPI",
+                "url": "https://www.bing.com/search?q=aol&FORM=USBAPI",
```

Comment:
so this will not impact doc generation, right?

Reply:
No it won't, as the semantic meaning of the string keeps unchanged.

Reply:
But is this what the service is returning, or is it changing what the actual response looks like?

Reply:
It will not change the response as it's the same string after parsing.